portal-web/test/functional/com/liferay/portalweb/macros/headless/objects/ObjectDefinitionAPI.macro

```diff
@@ -217,6 +217,21 @@ definition {
 		return "${objectFirstname}";
 	}
 
+	macro _getObjectsWithAggregationTerms {
+		Variables.assertDefined(parameterList = "${facetCriteria}");
```

Comment:
the parameter we're setting is aggregationTerms right? facetCriteria is something we get in a response of the body